pages/base/templatechecker.html

```diff
@@ -0,0 +1,29 @@
+/**
+* -----------------------------------------------------------------------------
+* @package     smartVISU
+* @author      Thomas Ernst
+* @copyright   2016
+* @license     GPL [http://www.gnu.de]
+* -----------------------------------------------------------------------------
+*/
+
+
+{% extends "system.html" %}
+
+{% block content %}
+<link rel="stylesheet" href="/lib/templatechecker/templatechecker.css">
```

Comment:
should be `<link rel="stylesheet" href="lib/templatechecker/templatechecker.css">`